### PR TITLE
vienna

### PR DESCRIPTION
**Pull request by smriad**

### CODE REVIEW

**Comment by BaseMax** [comment]
I presented a talk at the University of Vienna, Austria. That is the same city?

**Comment by BaseMax** [comment]
They have beautiful piano music.
Rare.
I suggest.

**Comment by jbampton** [comment]
@BaseMax yes Vienna, Austria 👍

**Comment by jbampton** [comment]
https://slurp--1847.org.readthedocs.build/en/1847/vienna.html

Screen Shot 2022-10-01 at 6 05 03 am

File: doc/source/vienna.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+The Hofburg Palace is located in the historic center and was the
+residence of the Habsburg royal family for more than six centuries. You
+will be able to know the perfectly furnished and decorated imperial
+rooms. In this castle, a large part of the history of Vienna is
+concentrated in its many rooms, including the Imperial Chapel, the
+Church, the Treasure Chamber and the Sisi Museum, where you will learn a
+little about the history of this controversial character.
+
+![Hofburg Palace](_static/images/vienna/image2.jpg){width="3.9178543307086615in"
+height="1.9944017935258094in"}
```
Comment (jbampton) [inline doc/source/vienna.md:45]:
```suggestion
![Hofburg Palace](_static/images/vienna/image2.jpg)
```

File: doc/source/vienna.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+The Austrian National Library is located in the Hofburg Palace complex,
+it was built in the 18th century in the Baroque style and houses more
+than 8 million documents, including papyri, old books and maps, on its
+tall wooden shelves. Its great Imperial Hall is really amazing, since in
+addition to countless old books there are beautiful frescoes in its
+dome, several marble statues and a couple of fantastic globes, one
+terrestrial and one celestial.
+
+![Austrian National Library](_static/images/vienna/image3.jpg){width="3.4241283902012247in"
+height="2.2737587489063866in"}
```
Comment (jbampton) [inline doc/source/vienna.md:58]:
```suggestion
![Austrian National Library](_static/images/vienna/image3.jpg)
```

File: doc/source/vienna.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+it is a Baroque-style palace with three-tiered gardens and large
+fountains. It is made up of two emblematic buildings: the Upper and
+Lower Belvedere. In the Upper Belvedere is the famous work of Gustav
+Klimt \"The Kiss\" and the Marble Hall, among his most outstanding works
+and in the Lower Belvedere you can marvel at the Hall of Mirrors, the
+Hall of Grotesques, the Gallery de Mármol and La Orangery (a beautiful
+greenhouse).
+
+![Belvedere Palace](_static/images/vienna/image4.jpg){width="3.9504943132108488in"
+height="2.2182042869641294in"}
```
Comment (jbampton) [inline doc/source/vienna.md:140]:
```suggestion
![Belvedere Palace](_static/images/vienna/image4.jpg)
```

**Review by jbampton — CHANGES_REQUESTED** [review]

**Review by jbampton — APPROVED** [review]

**Pull request merged**